### PR TITLE
OCPBUG8885: Added  'available' definition in 'understanding how to use PDBs' section.

### PR DESCRIPTION
**Pull request by subhtk**
Version(s): 4.10+

Issue: [OCPBUG8885](https://issues.redhat.com/browse/OCPBUGS-8885)

Link to docs preview:
[Docs Preview](https://59241--docspreview.netlify.app/openshift-enterprise/latest/nodes/pods/nodes-pods-configuring.html#nodes-pods-configuring-pod-distruption-about_nodes-pods-configuring)

QE review:

- [ x ]  QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/nodes-pods-pod-disruption-about.adoc [hunk 1]
```diff
@@ -30,6 +30,8 @@ parts:
 
 [NOTE]
 ====
+`Available` refers to the number of pods that are passing the readiness probes.
```
Comment (wking) [inline modules/nodes-pods-pod-disruption-about.adoc:33]: not all pods define readiness probes.  Is the actual definition pods that are declaring the `Ready=True` condition?

Comment (sunilcio) [reply modules/nodes-pods-pod-disruption-about.adoc:33]: Yes, that is correct. It should be defined as pods that have condition Ready=True

Comment (subhtk) [reply modules/nodes-pods-pod-disruption-about.adoc:33]: As per the conversation, I will update the doc.

Comment (sdodson) [reply modules/nodes-pods-pod-disruption-about.adoc]: Can we accurately state what's considered in determining when a pod is `Ready=True` ?

**Comment by kalexand-rh** [comment]
@sunilcio, is this content correct now?

File: modules/nodes-pods-pod-disruption-about.adoc (hunk 1 above)
Comment (subhtk) [reply modules/nodes-pods-pod-disruption-about.adoc]: @sdodson can you confirm the above comment? If that suffices I would make the changes accordingly.

Comment (sdodson) [reply modules/nodes-pods-pod-disruption-about.adoc]: Sure that's fine.

Comment (subhtk) [reply modules/nodes-pods-pod-disruption-about.adoc]: Great, I will make the changes accordingly.

**Comment by sdodson** [comment]
/lgtm
I think there's a risk it's still not clear enough what's considered in defining whether a pod is `Ready=True` but understanding that likely entails reading this entire page as there's no succinct answer given there.

https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#:~:text=A%20Pod%20is%20considered%20ready,a%20container%20application%20has%20started.

**Comment by kelbrown20** [comment]
Secondary peer review: Updates look great!
/lgtm

**Review by sheriff-rh — COMMENTED** [review]
LGTM, merging.

**Pull request merged**